tests/unit/models/transactions/test_transaction.py

```diff
@@ -79,3 +80,44 @@ def test_to_dict_flag_list(self):
         expected_flags = 0b111
         value = tx.to_dict()["flags"]
         self.assertEqual(value, expected_flags)
+
+    def test_to_dict_ticket_sequence(self):
+        tx = Transaction(
+            account=_ACCOUNT,
+            fee=_FEE,
+            ticket_sequence=_TICKET_SEQUENCE,
+            transaction_type=TransactionType.ACCOUNT_DELETE,
+        )
+        value = tx.to_dict()["ticket_sequence"]
+        self.assertEqual(value, _TICKET_SEQUENCE)
+
+    def test_to_dict_ticket_sequence_with_sequence_zero(self):
```

Comment:
Basic intent with this suggestion is to make it really obvious to future readers exactly what this test should do, especially if they have no context on why this test exists. 
```suggestion
    def test_throws_when_ticket_sequence_and_sequence_both_non_zero(self):
```